.github/workflows/ros1.yml

```diff
@@ -56,6 +56,37 @@ jobs:
         ros-distro: ${{ matrix.distro }}
         gazebo-version: ${{ matrix.gazebo }}
         workspace-dir: simulation_ws
+     - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_ROS1 }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY_ROS1 }}
+        aws-region: ${{ secrets.AWS_REGION }}
+      if: github.event_name == 'schedule' && github.ref == 'refs/heads/ros1'
+    - id: deploy_bundle
+      name: Deploy bundle to S3
+      uses: aws-robomaker-github-actions/s3-cp-action@v2.1.1
```

Comment:
Shouldn't this be `aws-robotics/aws-robomaker-github-actions/s3-cp-action@2.1.1`?